pkg/config/schema/collections/collections.gen.go

```diff
@@ -232,6 +233,7 @@ var (
 		Version:    "v1beta1",
```

Comment:
The primary version is the one we use throughout the code. If we switch to v1, then users MUST be on v1, which is new. Generally we give it a few releases, then switch to the newer versions, so people have time to catch up. Since we do not ship the CRDs ourselves, we cannot control their version.

Reply:
No, because they may not have v1 at all so read calls to api server will get a 404